services/webfinger/pkg/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"context"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+)
+
+// Config combines all available configuration parts.
+type Config struct {
+	Commons *shared.Commons `yaml:"-"` // don't use this directly as configuration for a service
+
+	Service Service `yaml:"-"`
+
+	Tracing *Tracing `yaml:"tracing"`
+	Log     *Log     `yaml:"log"`
+	Debug   Debug    `yaml:"debug"`
+
+	HTTP HTTP `yaml:"http"`
+
+	Instances []Instance `yaml:"instances"`
+	Relations []string   `yaml:"relations" env:"WEBFINGER_RELATIONS" desc:"A comma-separated list of relation URIs or registered relation types to add to webfinger responses."`
+	IDP       string     `yaml:"idp" env:"OCIS_URL;OCIS_OIDC_ISSUER;WEBFINGER_OIDC_ISSUER" desc:"The identity provider href for the openid-discovery relation."`
+	OcisURL   string     `yaml:"ocis_url" env:"OCIS_URL;WEBFINGER_OCIS_URL" desc:"URL, where oCIS is reachable for users."`
```

Comment:
```suggestion
	OcisURL   string     `yaml:"ocis_url" env:"OCIS_URL;WEBFINGER_OCIS_URL" desc:"The public facing URL of the oCIS frontend."`
```
This is the string we use for the `OCIS_URL`

Reply:
hm this is not the ocis frontend. this is more about the ocis api ...

in theory admins could deploy web at eg. `https://drive.ocis.test` and the webdav, ocs, libregraph, ... APIs at `https://shard-a-k.drive.ocis.test` as well as `https://shard-l-z.drive.ocis.test`. The webfinger server **could** return two ocis instance relations and a yet to be defined ocis web relation.

Reply:
Note that whenever OCIS_URL was added in one service, that description was used. If we want to _add_ info, I am fine with, but we should not confuse the reader with different basic descriptions. Just my 2c.